cfme/tests/test_manual.py

```diff
@@ -13393,6 +13430,7 @@ def test_infrastructure_provider_left_panel_titles():
         casecomponent: Infra
         caseimportance: low
         initialEstimate: 1/18h
+        caseautomation: manualonly
```

Comment:
With the new pytest-polarion-collect, all the other cases should have the caseautomation value you want with the existing marker.

This test case you will have to add `@pytest.mark.manual('manualonly')` to get 'manualonly' caseautomation value here via the marker.

Reply:
@mshriver Okay, added manualonly arg to those tests which cannot be automated.